astro/src/content/docs/apis/_application-response-body-base.mdx

```diff
@@ -8,7 +8,6 @@ import RemovedSince from 'src/components/api/RemovedSince.astro';
 import ApplicationOauthconfigurationClientauthenticationpolicy from 'src/content/docs/apis/_application-oauthConfiguration-clientAuthenticationPolicy.mdx';
 import TenantApplicationEmailConfiguration from 'src/content/docs/apis/_tenant-application-email-configuration.mdx';
 import TenantApplicationPhoneConfiguration from 'src/content/docs/apis/_tenant-application-phone-configuration.mdx';
-import JwtAlgorithm from 'src/content/docs/apis/_jwt_algorithm.mdx';
```

Comment:
This component was imported here and one other spot but not used on the page. I deleted it.